pkg/pool-cow/contracts/CowRouter.sol

```diff
@@ -28,11 +28,18 @@ contract CowRouter is SingletonAuthentication, VaultGuard, ICowRouter {
     // Protocol fee percentage capped at 10%.
     uint256 internal constant _MAX_PROTOCOL_FEE_PERCENTAGE = 10e16;
 
+    // Address that will receive the protocol fees on withdraw.
+    address internal _feeSweeper;
     uint256 internal _protocolFeePercentage;
     mapping(IERC20 => uint256) internal _protocolFees;
 
-    constructor(IVault vault, uint256 protocolFeePercentage) VaultGuard(vault) SingletonAuthentication(vault) {
+    constructor(
+        IVault vault,
+        uint256 protocolFeePercentage,
+        address feeSweeper
+    ) VaultGuard(vault) SingletonAuthentication(vault) {
         _protocolFeePercentage = protocolFeePercentage;
+        _feeSweeper = feeSweeper;
```

Comment:
Since there is validation and an event tied to this, would use the pattern of making a `_setFeeSweeper` and calling it here and from the external `setFeeSweeper`.

Same thing with the protocol fee percentage: `_setProtocolFeePercentage` called from both places.